templates/login.html

```diff
@@ -1,14 +1,10 @@
 {% extends "base.html" %}
 {% load bootstrap4 %}
-{% block title %}
-Login Here
-{% endblock %}
-
 {% block content %}
 <h2>Log In</h2>
 <form method="post">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <button type="submit">Log In</button>
+ {% csrf_token %}
+ {% bootstrap_form form %}
```

Comment:
There are some other places where you can make this same change, but no need to do that now if you don't want to.